src/snowcli/connection_config.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+import os
+
+from configparser import ConfigParser
+
+from snowcli.exception import EnvironmentVariableNotFoundError
+
+
+class ConnectionConfigs:
+    def __init__(self, snowsql_config_path="~/.snowsql/config"):
+        self.snowsql_config_path = snowsql_config_path
+        self._config = ConfigParser(inline_comment_prefixes="#", interpolation=None)
+        self._config.read(os.path.expanduser(snowsql_config_path))
+
+    def is_connection_exists(self, connection_name):
```

Comment:
```suggestion
    def connection_exists(self, connection_name: str):
```
WDYT?

Reply:
I'm for it, but we should decide to add types everywhere or not